DisableJIT linker warning (unused obj file)

`AsmJsLink.obj : warning LNK4221: This object file does not define any previously undefined public symbols, so it will not be used by any link operation that consumes this library [D:\j\workspace\daily_disable---1fca0551\lib\Runtime\Language\Chakra.Runtime.Language.vcxproj]`

See e.g. http://dotnet-ci.cloudapp.net/job/Microsoft_ChakraCore/job/master/job/daily_disablejit_x86_debug/98/consoleText (find on page: LNK) 

See also: https://github.com/Microsoft/ChakraCore/wiki/Build-Status#disablejit-builds, https://github.com/Microsoft/ChakraCore/wiki/Build-Status-%28linux%29#disablejit-builds

FYI @digitalinfinity @MikeHolman @Cellule 
cc @aneeshdk

--- a/lib/Runtime/Language/Chakra.Runtime.Language.vcxproj
+++ b/lib/Runtime/Language/Chakra.Runtime.Language.vcxproj
@@ -70,7 +70,9 @@
     </ClCompile>
     <ClCompile Include="$(MSBuildThisFileDirectory)AsmJs.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)AsmJsByteCodeGenerator.cpp" />
-    <ClCompile Include="$(MSBuildThisFileDirectory)AsmJsLink.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)AsmJsLink.cpp">
+      <ExcludedFromBuild Condition="'$(BuildJIT)'=='false'">true</ExcludedFromBuild>
+    </ClCompile>
     <ClCompile Include="$(MSBuildThisFileDirectory)AsmJsModule.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)AsmJsTypes.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)AsmJsUtils.cpp" />
@@ -184,7 +186,9 @@
       <ExcludedFromBuild Condition="'$(Platform)'!='Win32' AND '$(Platform)'!='x64'">true</ExcludedFromBuild>
     </ClInclude>
     <ClInclude Include="AsmJsJitTemplate.h" />
-    <ClInclude Include="AsmJsLink.h" />
+    <ClInclude Include="AsmJsLink.h">
+      <ExcludedFromBuild Condition="'$(BuildJIT)'=='false'">true</ExcludedFromBuild>
+    </ClInclude>
     <ClInclude Include="AsmJsModule.h" />
     <ClInclude Include="AsmJsBuiltInNames.h" />
     <ClInclude Include="AsmJsTypes.h" />

--- a/test/rlexedirs.xml
+++ b/test/rlexedirs.xml
@@ -380,13 +380,15 @@
 <dir>
   <default>
     <files>TTExecuteBasic</files>
-    <tags>sequential,exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized</tags>
+    <!-- TODO Investigate whether these tests really belong under `require_backend` -->
+    <tags>sequential,exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized,require_backend</tags>
   </default>
 </dir>
 <dir>
   <default>
     <files>TTBasic</files>
-    <tags>sequential,exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized</tags>
+    <!-- TODO Investigate whether these tests really belong under `require_backend` -->
+    <tags>sequential,exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized,require_backend</tags>
   </default>
 </dir>
 </regress-exe>

